Source/ACE/Entity/Player.cs

```diff
@@ -765,15 +770,96 @@ public void SaveOptions()
         }
 
         /// <summary>
+        /// Set the currenly position of the character, to later save in the database.
+        /// </summary>
+        public void SetPhysicalCharacterPosition()
+        {
+            // Saves the current player position after converting from a Position Object, to a CharacterPosition object
+            character.SetCharacterPositions(CharacterPositionType.PhysicalLocation, CharacterPositionExtensions.positionToCharacterPosition(character.Id, Session.Player.Position, CharacterPositionType.PhysicalLocation));
+        }
+
+        /// <summary>
+        /// Set's the active lifestone for use when recalling "@ls or /lifestone"
+        /// </summary>
+        public void SetLifestoneUseCharacterPosition()
+        {
+            // Saves the current player position after converting from a Position Object, to a CharacterPosition object
+            character.SetCharacterPositions(CharacterPositionType.LifestoneUsed, CharacterPositionExtensions.positionToCharacterPosition(character.Id, Session.Player.Position, CharacterPositionType.LifestoneUsed));
+        }
+
+        /// <summary>
+        /// Set's the tied lifestone for use casting the Lifestone Recall spell
+        /// </summary>
+        public void SetLifestoneTiedCharacterPosition(CharacterPosition newPosition)

```

Comment:
It seems like we should have a single "SetPosition(Enum.Position positionType, Position position)" method so that we don't have to have 1 of these for each.